airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -126,6 +126,23 @@ def docker_runner_fixture(image_tag, tmp_path) -> ConnectorRunner:
     return ConnectorRunner(image_tag, volume=tmp_path)
 
 
+@pytest.fixture(name="previous_connector_image_name")
+def previous_connector_image_name_fixture(image_tag, inputs) -> str:
+    """Fixture with previous connector image name to use for backward compatibilty tests"""
+    return f"{image_tag.split(':')[0]}:{getattr(inputs, 'previous_connector_version')}"
+
+
+@pytest.fixture(name="previous_connector_docker_runner")
+def previous_connector_docker_runner_fixture(previous_connector_image_name, tmp_path) -> ConnectorRunner:
+    try:
+        return ConnectorRunner(previous_connector_image_name, volume=tmp_path / "previous_connector")
+    except (errors.NotFound, errors.ImageNotFound):
+        logging.warning(
+            f"\n We did not find the {previous_connector_image_name} image for this connector. This probably means this version has not yet been published to our docker registry."
```

Comment:
voicing: i try to avoid words like `our` since this is an open source product meant to interoperate in many environments and used by different people
```suggestion
            f"\n We did not find the {previous_connector_image_name} image for this connector. This probably means this version has not yet been published to an accessible docker registry like dockerhub."
```